apps/src/javalab/JavabuilderConnection.js

```diff
@@ -36,50 +35,73 @@ export default class JavabuilderConnection {
     this.setIsTesting = setIsTesting;
     this.executionType = executionType;
     this.miniAppType = miniAppType;
-    this.overrideSources = overrideSources;
   }
 
   // Get the access token to connect to javabuilder and then open the websocket connection.
   // The token prevents access to our javabuilder AWS execution environment by un-verified users.
```

Comment:
nit: mind adding to this comment describing when we'd use this as opposed to the other functions? just want to make sure we document that this is meant for typical student code execution vs the other two are special cases